lib/rust/ensogl/example/render-profile-flamegraph/src/lib.rs

```diff
@@ -89,6 +90,9 @@ async fn get_data() -> String {
     let response = wasm_bindgen_futures::JsFuture::from(response).await.unwrap();
     assert!(response.is_instance_of::<web_sys::Response>());
     let response: web_sys::Response = response.dyn_into().unwrap();
+    if !response.ok() {
+        return enso_profiler_demo_data::create_data().await;
```

Comment:
Same here. Some logging could help to indicate what is going on to someone looking at the debug scene and dev console.